charts/sbombastic/templates/worker/deployment.yaml

```diff
@@ -28,6 +28,12 @@ spec:
           args:
             - -nats-url
             - {{ .Release.Name }}-nats.{{ .Release.Namespace }}.svc.cluster.local:4222
+            {{- if .Values.worker.trivyDBRepository }}
+            - -trivy-db-repository={{ .Values.worker.trivyDBRepository }}
```

Comment:
I think it would be safer to [quote](https://helm.sh/docs/chart_template_guide/functions_and_pipelines/) the input given by the user here